### DISCUSSION TITLE
Re-Applying for acedemic discount cupon for organization

### DISCUSSION THREAD

**Discussion by Achmiel4** [community/community#54385] [GitHub Education]
### Hi everyone,

Recently my organization's ([Illini Solar Car](https://github.com/IlliniSolarCar)) academic discount coupon for github team has expired and I am having trouble finding the correct way to re-apply. We have used this discount for years and in the past have had no issues re-applying.

I reached out to support through a ticket but was instructed to ask for help here. Could someone help me re-apply?

**Comment by benjamid** [comment]
I am having about the same issue for my orgs as well. I found that I can re-upgrade them here: https://education.github.com/globalcampus/teacher

However, when I do they are immediately expired the next day, so that's not much help. You can see the behavior here: 
![Github Education Account Issue_DeIdent](https://github.com/user-attachments/assets/f8c1d11e-bcd0-42d7-8c89-416afa8975b4)

This has to be a bug, because I can't imagine this is intended behavior to without warning downgrade all the organizations that a faculty member created, and then let you upgrade them, then immediately downgrade them.
